lbrynet/core/LBRYWallet.py

```diff
@@ -520,19 +558,13 @@ def _open_db(self):
                                 "    txid text, " +
                                 "    sd_hash text)")
 
-    def _save_name_metadata(self, name, sd_hash, txid):
-        d = self.db.runQuery("select * from name_metadata where txid=?", (txid,))
+    def _save_name_metadata(self, name, txid, sd_hash):
+        d = self.db.runQuery("select * from name_metadata where name=? and txid=? and sd_hash=?", (name, txid, sd_hash))
```

Comment:
You can have multiple claims in the same tx. None of the keys are assuredly unique, but the set of all three is.